### ISSUE TITLE
/ is not needed when favicons placed in root

### ISSUE THREAD

**Issue by slavap** [RealFaviconGenerator/realfavicongenerator#167]
Jetty and Tomcat8 on Windows7. I have to remove all starting / from index.html, manifest.json, and browserconfig.xml, i.e. /apple-touch-icon-57x57.png changed to apple-touch-icon-57x57.png and so on for all icons. After that it works just fine.

**Comment by phbernard** [comment]
Really? What goes wrong with the leading `/`?

**Comment by slavap** [comment]
Yes. It's not working for me when there is leading /, so I have to manually remove it. It works fine after that.

**Comment by phbernard** [comment]
What happens when the leading slash is present? Maybe you tried to add your site to the homescreen of your Android phone and the icon does not show up? Or you tried something else?

What about the locations of your files? Since your original paths are like `/apple-touch-icon-57x57.png`, you should be able to access `http://yoursite.com/apple-touch-icon-57x57.png`. Does this work?

**Comment by slavap** [comment]
After your question I've realized, that my problem probably is related to deploy path of my app. So the path is https://har:8443/myapp/
And the right path for my case is /myapp/ instead of /, right?

Also the only, unrelated to this issue, problem which I still have, is "Add to homescreen" function on iOS 8.3. If I host my app over http - I'm getting my icon, but when hosted over https - standard ugly icon.

**Comment by phbernard** [comment]
That's right, if you app is mapped on `https://har:8443/myapp/`, the path to your icons should be `/myapp`. As a general rule, I consider a good practice to look at the source of the page (via the web browser, not your actual source code) and check that the path to one of your icon works.

As far as I know, iOS can deal with the Apple Touch icon over HTTPS. Could there be a path issue?

**Comment by slavap** [comment]
Thank you!
No, it's definitely not a host path problem :-(
http://har:8080/myapp/ works just fine and gives me nice icon on homescreen, but
https://har:8443/myapp/ gives standard ugly icon.
Content of both deployments are the same.

**Comment by phbernard** [comment]
I never encountered the "HTTPS+Specific port number" combo. Additional question: is your certificate signed by an authority or it is self-signed?

**Comment by slavap** [comment]
My certificate is signed by authority. I will try to eliminate explicit port number for https, but really doubt that will help. On Android Chrome both http and https are working just fine, so it's kind of iOS/Safari specific problem.
